sdk/go/pulumi/context.go

```diff
@@ -2129,6 +2131,11 @@ func (ctx *Context) RegisterStackTransform(t ResourceTransform) error {
 	return err
 }
 
+// RegisterStackTransform adds a transform to all future resources constructed in this Pulumi stack.
+func (ctx *Context) RegisterStackTransform(t ResourceTransform) error {
```

Comment:
No `Deprecated` here?

Reply:
I added it to `RegisterStackTransformation` instead :facepalm: 

Fixing that now :heavy_check_mark: